### PR TITLE
The Great Reform Of Styles

### LINKED ISSUE #34: 💅🏼 style the fuck up
most of the styles are really basic. so we need to improve it a bit.

also, most of the class names are so fucked up, I can't even pick the words to describe. so it should be fixed as well

### PR DESCRIPTION
**Pull request by BUI-studies**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-  <body>
-    <div id="app"></div>
-    <script type="module" src="./main.js"></script>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="./vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Vite App</title>
```
Comment (ohiienko-r) [inline index.html]: Shoudn't the app name be "Kurkulyator"?

**Review by AKot1988 — COMMENTED** [review]
Owerall looking good. i've seen that you have changed all scss files names

**Review by ohiienko-r — APPROVED** [review]
Now everything looks much cleaner and readable! Like your code sty;e and approach. LGTM

**Review by ohiienko-r — APPROVED** [review]
Approved

**Review by AKot1988 — COMMENTED** [review]
approved

**Pull request merged**